optionals/cinematicSpec/functions/fn_api_cancelCinematicSpec.sqf

```diff
@@ -0,0 +1,37 @@
+/**
+ * if player is in cinematicSpec, send them back to the spectator mode they came from.
+ * this function is callbacked. example usage:
+ * [{
+ *     [false] call ace_spectator_fnc_setSpectator;
+ * }] call grad_cinematicSpec_fnc_cancelCinematicSpec;
+ *
+ * returns "sync" on immediate callback execution and "async" if callback execution is delayed
+ */
+
+#include "script_component.hpp"
+
+private _callback = param [0, {}];
```

Comment:
Params? :u

Reply:
I *think* I'm not using params becaus @kayler-renslow's IntelliJ plugin wont recognize the variable declarations.

Reply:
und schneller :u

Reply:
und macht alle params automatisch private